lib/api.js

```diff
@@ -164,6 +165,11 @@ PlexAPI.prototype._request = function _request(options) {
 
         resolveValue = body;
 
+        if (response.statusCode === 403 && !options.overrideToken && self.authToken) {
```

Comment:
In a new round of testing I *finally* got 403 responses from the server 👯 

> Alternatively I suppose the code could be throw a different error and the user of this client could update the token and try again but if this is a common case and we have both tokens I figured why not just try the same operation again with the auth token?

Falling back to the `authToken` seems strange though. The `authToken` originates from the main/master user, which usually has access to everything on the server, agreed? What I've observed so far, is that when a managed user does *not* have access to perform a request, the same request is performed with `authToken` instead, which at least in my configuration always passes because it's the master user's token. That means the security restrictions set on a managed user is actually bypassed. I my mind that doesn't seem like a good idea?